jacoco-maven-plugin/src/org/jacoco/maven/CheckMojo.java

```diff
@@ -161,8 +161,8 @@ private boolean canCheckCoverage() {
 	}
 
 	@Override
-	public void executeMojo() throws MojoExecutionException,
-			MojoExecutionException {
```

Comment:
Duplicate in throws clause - [fixed in master](https://github.com/jacoco/jacoco/commit/b71209adbf45b454cdf2ebda6f144f2687430c9e).